src/helpers.js

```diff
@@ -7,7 +7,7 @@ import { lazy } from 'yup';
 
 import { supportedTypes, getInputType } from './internals/fields';
 import { pickXKey } from './internals/helpers';
-import { containsHTML, hasProperty, wrapWithSpan } from './utils';
```

Comment:
Wondering if we should delete containsHTML and wrapWithSpan as they're not used anymore

Reply:
Yes, let's remove both of them for now! Perhaps it was used in the past, but with the reactors to open source this it got forgotten.

Reply:
Removed!